src/libraries/System.Private.Xml/src/System/Xml/Schema/NamespaceList.cs

```diff
@@ -363,11 +363,11 @@ public static NamespaceList Intersection(NamespaceList o1, NamespaceList o2, boo
                 }
                 if (!v1Compat)
                 {
-                    if (o1._targetNamespace == string.Empty)
+                    if (o1._targetNamespace.Length == 0)
                     { // clause 6 - o1 is negation of absent
                         nslist = o2.Clone();
                     }
-                    else if (o2._targetNamespace == string.Empty)
+                    else if (o2._targetNamespace.Length == 0)
```

Comment:
Same question for these; how do we know they're non-null?

Reply:
```suggestion
                    else if (o2._targetNamespace == string.Empty)
```